apps/mobile/src/features/activity/activity-empty.tsx

```diff
@@ -19,7 +19,7 @@ export function ActivityEmpty() {
         <Text textAlign="center" variant="label01">
           {t({
             id: 'activity-empty.title',
-            message: 'Make your first transaction to get started',
```

Comment:
I had to change this text but I can't see it being taken by `lingui compile` so it wasn't updating properly